### DISCUSSION TITLE
mariadb-1  Access denied for user to database

### DISCUSSION THREAD

**Discussion by artincube** [photoprism/photoprism#4355] [Q&A]
I updated Photoprism after some period of inactivity, and also updated MariaDB from 10 to 11.
At first I modified the original old compose file (based on 3.5 if I remember well) to work with 11 (including the command mod).
I managed to make it work.

Then I pulled the docker compose file for Mac (I am on Mac M1) and modified it with all parameters from original old file.
Now I get that error in the logs:
`mariadb-1     | 2024-06-24 23:22:09 6 [Warning] Access denied for user 'photoprism'@'%' to database 'photoprism_v2'`
I go back to old compose file and all works well.
I cannot see what in that new compose file is making the problem.

Attached are both.
Thank you.

[composefiles.zip](https://github.com/user-attachments/files/15962285/composefiles.zip)

**Comment by omerdduran** [comment]
Hi! Although this is a late reply, I hope it can help others facing similar issues. First, I recommend checking PhotoPrism's troubleshooting checklist: https://docs.photoprism.app/getting-started/troubleshooting/mariadb/#cannot-connect

If the troubleshooting guide doesn't resolve your issue, there is a [solution](https://github.com/photoprism/photoprism/issues/67#issuecomment-753069778) that has worked for other users.
